src/mame/drivers/dynax.cpp

```diff
@@ -1093,8 +1095,12 @@ uint8_t dynax_state::tenkai_ip_r(offs_t offset)
 
 				// player 1
 				case 0x82:
-					if (m_keyb >= 5)
+					if (m_input_mode != 0xFF)
+						return 0x00;
+					if (m_keyb >= 5){
 						logerror("%04x: unmapped keyb=%02x read\n", m_maincpu->pc(), m_keyb);
+						return 0x00;
+					}
```

Comment:
Please follow the style of the surrounding code.  Use Allman brace positioning and lowercase letters in hexadecimal literals.